modules/govuk_jenkins/templates/jobs/deploy_lambda_app.yaml.erb

```diff
@@ -0,0 +1,51 @@
+---
+- scm:
+    name: govuk-lambda-app-deployment_Deploy_Lambda_App
+    scm:
+        - git:
+            url: git@github.com:alphagov/govuk-lambda-app-deployment.git
+            branches:
+              - master
+            wipe-workspace: true
+
+- job:
+    name: Deploy_Lambda_App
+    display-name: Deploy_Lambda_App
+    project-type: freestyle
+    description: "This job uploads an AWS Lambda app to an S3 bucket and kicks off the relevant Terraform job"
+    properties:
+        - github:
+            url: https://github.com/alphagov/govuk-lambda-app-deployment
+    scm:
+      - govuk-lambda-app-deployment_Deploy_Lambda_App
+    builders:
+      - shell: |
+          cd $TARGET_APPLICATION
+          ./jenkins.sh # this job should create the source ready for uploading to an S3 bucket
+          s3cmd sync . s3://govuk-lambda-applications-$ENVIRONMENT/$TARGET_APPLICATION/
```

Comment:
If we use `aws s3api put-object` here it returns the version id. It's in JSON though... e.g.

```
{
    "VersionId": "HdrY9L9wuhSA1z0d.Xc1h05F3zeigeYM",
    "ETag": "\"814c08483064b7d0da54ce1512a5d33a\""
}
```